modules/platforms/dotnet/Apache.Ignite/IgniteClientConfiguration.cs

```diff
@@ -169,5 +170,12 @@ public IgniteClientConfiguration(IgniteClientConfiguration other)
         /// See <see cref="SslStreamFactory"/>.
         /// </summary>
         public ISslStreamFactory? SslStreamFactory { get; set; }
+
+        /// <summary>
+        /// Gets or sets the authenticator.
```

Comment:
Probably should add that if it's `null` then auth is not used.

Reply:
Added.